scripts/workarounds/boot-order/README.md

```diff
@@ -0,0 +1,47 @@
+# Boot Order Workaround
+
+This directory includes the script and the library necessary for applying the workaround to all reachable NCNs.
+
+## Usage
+
+1. Run `run.sh`.
+
+    ```bash
+    ncn# /usr/share/doc/csm/scripts/workarounds/boot-order/run.sh
+    ```
+
+   Example output:
+
+   ```text
+   Uploading new metal-lib.sh to ncn-m001:/srv/cray/scripts/metal/metal-lib.sh ... Done
+   Uploading new metal-lib.sh to ncn-m002:/srv/cray/scripts/metal/metal-lib.sh ... Done
+   Uploading new metal-lib.sh to ncn-m003:/srv/cray/scripts/metal/metal-lib.sh ... Done
+   Uploading new metal-lib.sh to ncn-s001:/srv/cray/scripts/metal/metal-lib.sh ... Done
+   Uploading new metal-lib.sh to ncn-s002:/srv/cray/scripts/metal/metal-lib.sh ... Done
+   Uploading new metal-lib.sh to ncn-s003:/srv/cray/scripts/metal/metal-lib.sh ... Done
+   Uploading new metal-lib.sh to ncn-s004:/srv/cray/scripts/metal/metal-lib.sh ... Done
+   Uploading new metal-lib.sh to ncn-w001:/srv/cray/scripts/metal/metal-lib.sh ... Done
+   Uploading new metal-lib.sh to ncn-w002:/srv/cray/scripts/metal/metal-lib.sh ... Done
+   Uploading new metal-lib.sh to ncn-w003:/srv/cray/scripts/metal/metal-lib.sh ... Done
+   Uploading new metal-lib.sh to ncn-w004:/srv/cray/scripts/metal/metal-lib.sh ... Done
+   Refreshing the bootorder on [11] NCNs ... Done
+   The following NCNs contain the boot-order patch:
+   ncn-m001
+   ncn-m002
+   ncn-m003
+   ncn-s001
+   ncn-s002
+   ncn-s003
+   ncn-s004
+   ncn-w001
+   ncn-w002
+   ncn-w003
+   ncn-w004
+   This workaround has completed.
+   ```
+
+## Origin
```

Comment:
Is this useful/relevant information for our customer-facing CSM product documentation?

Reply:
Script is copied directly from 1.3/1.4, so I want the diff to be 0 and have a link to the origin like the kdump workaround.